mentat/streaming_printer.py

```diff
@@ -13,7 +14,22 @@ def __init__(self):
         self.finishing = False
         self.shutdown = False
 
-    def add_string(self, string: str, end: str = "\n", color: str | None = None):
+    def add_string(
+        self,
+        string: str,
+        end: str = "\n",
+        color: str | None = None,
+        style: str | None = None,
+    ):
+        if style is not None:
+            session_context = SESSION_CONTEXT.get()
+            if session_context.config.theme is not None:
+                theme = Themes.get(session_context.config.theme)
+            else:
+                theme = None
+            if theme is not None:
+                color = theme[style]
+
```

Comment:
Could we just pass on the style here instead of converting to the color?

Looks like these strings end up being passed to `stream.send` eventually, which should take care of this.

Reply:
We could but the next function (in the same file) seems to be very low level tied to colors.